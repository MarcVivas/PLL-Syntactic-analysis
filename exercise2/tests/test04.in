// Test invalid expressions
1 &&;
aguacate;
2 * (3 + );
?;
?;

// Correct expressions
+3;
-4;
-4 + 3;
