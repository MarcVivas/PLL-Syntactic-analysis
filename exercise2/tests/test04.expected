Error at line 2: syntax error
Error at line 2: Invalid expression
Error at line 3: syntax error
Error at line 3: Invalid expression
Error at line 4: syntax error
Error at line 4: Invalid expression
Error at line 5: syntax error
Error at line 5: Invalid expression
Error at line 6: syntax error
Error at line 6: Invalid expression
3
-4
-1
